var/spack/repos/builtin/packages/geopm-service/package.py

```diff
@@ -0,0 +1,148 @@
+    conflicts("target=ppc64le:", msg="Only available on x86_64")
+
+    patch("0001-Support-NVML-via-CUDA-installation.patch", when="+nvml")
+
+    # Autotools dependencies
+    depends_on("automake", type="build")
+    depends_on("libtool", type="build")
+    depends_on("file")
+
+    # Docs dependencies
+    depends_on("doxygen", type="build", when="+docs")
+    depends_on("py-docstring-parser@0.13.0:", type="build", when="+docs")
+    depends_on("py-sphinx@4.5:", type="build", when="+docs")
+    depends_on("py-sphinx-rtd-theme@1:", type="build", when="+docs")
+    depends_on("py-sphinx-emoji@0.2.0:", type="build", when="+docs")
+    depends_on("py-sphinx-tabs@3.3.1:", type="build", when="+docs")
+    depends_on("py-pygments@2.13.0:", type="build", when="+docs")
+
+    # Other Python dependencies
+    depends_on("python@3.6:3")
+    depends_on("py-dasbus@1.6.0:", type=("build", "run"), when="+systemd")
+    depends_on("py-cffi@1.14.5:", type="run")
+    depends_on("py-psutil@5.8.0:", type="run")
+    depends_on("py-jsonschema@3.2.0:", type="run")
+    depends_on("py-pyyaml@6.0:", type="run")
+    depends_on("py-setuptools@53.0.0:", type="build")
+
+    # Other dependencies
+    depends_on("bash-completion")
+    depends_on("unzip")
+    depends_on("libcap", when="+libcap")
+    depends_on("liburing", when="+liburing")
+    depends_on("oneapi-level-zero", when="+levelzero")
+    depends_on("cuda", when="+nvml")
+
+    configure_directory = "service"
+
+    def autoreconf(self, spec, prefix):
+        bash = which("bash")
+        with working_dir("service"):
+            bash("./autogen.sh")
+
+    def configure_args(self):
+        args = [
+            "--with-bash-completion-dir="
+            + join_path(self.spec.prefix, "share", "bash-completion", "completions")
+        ]
+
+        args += self.enable_or_disable("debug")
+        args += self.enable_or_disable("docs")
+        args += self.enable_or_disable("systemd")
+        args += self.enable_or_disable("liburing")
+        args += self.with_or_without("liburing", activation_value="prefix")
+        args += self.enable_or_disable("libcap")
+        args += self.with_or_without("gnu-ld")
+
+        args += self.enable_or_disable("levelzero")
+        args += self.enable_or_disable("nvml")
+        if "+nvml" in self.spec:
+            args += [
+                "--with-nvml=" + join_path(self.spec["cuda"].prefix, "targets", "x86_64-linux")
+            ]
+
+        args += self.enable_or_disable("rawmsr")
+        return args
+
+    def setup_run_environment(self, env):
+        if os.path.isdir(self.prefix.lib64):
+            lib_dir = self.prefix.lib64
+        else:
+            lib_dir = self.prefix.lib
+        env.prepend_path("LD_LIBRARY_PATH", lib_dir)
+
+        python_version = self.spec["python"].version.up_to(2)
+        python_inst_dir = join_path(lib_dir, "python{0}".format(python_version), "site-packages")
+        env.prepend_path("PYTHONPATH", python_inst_dir)
```

Comment:
If you add:
```python
extends("python")
```
all of this will be done automatically for you, and it will work even if the Python libs are installed to a different location.